### PR TITLE
Add hardhat-zksync-chai-matchers plugin

### PR DESCRIPTION
**Pull request by mpopovac-txfusion**

### CODE REVIEW

File: docs/api/hardhat/plugins.md [hunk 1]
```diff
@@ -277,3 +277,43 @@ For more details about the dockerized local setup, check out [Local testing](htt
...
+
+Add the latest version of this plugin to your project with the following command:
+
+```
+# Yarn
+yarn add -D @matterlabs/hardhat-zksync-chai-matchers
+
+# Npm
+npm i -D @matterlabs/hardhat-zksync-chai-matchers
+```
```
Comment (krofax) [inline docs/api/hardhat/plugins.md:295]: This doesn't seem to be live yet.

**Review by githubdoramon — APPROVED** [review]

File: docs/api/hardhat/plugins.md [hunk 2]
```diff
@@ -285,3 +285,113 @@ If network argument `--network` or `defaultNetwork` configuration are not specif
...
+```javascript
+await expect(contract.setAmount(100)).to.be.revertedWithCustomError(contract, 'InvalidAmount');
+```
+
+#### emit
+
+Assert that a transaction emits a specific event:
+
+```javascript
+await expect(contract.setAmount(100)).to.emit(token, 'AmountUpdated');
```
Comment (githubdoramon) [inline docs/api/hardhat/plugins.md:394]: I guess the "token" variable should be "contract" right?

**Review by githubdoramon — CHANGES_REQUESTED** [review]

**Review by githubdoramon — DISMISSED** [review]

**Comment by uF4No** [comment]
I think we also need to update the Hardhat Getting started page (/api/hardhat/getting-started.html) where it mentions: "zkSync has three plugins for Hardhat", as now we have 4 😉
Also, it'd be a good idea to also mention this plugin at the end in the Local testing page (api/hardhat/testing.html).

**Comment by githubdoramon** [comment]
> @mpopovac-txfusion I think we also need to update the Hardhat Getting started page (/api/hardhat/getting-started.html) where it mentions: "zkSync has three plugins for Hardhat", as now we have 4 😉 Also, it'd be a good idea to also mention this plugin at the end in the Local testing page (api/hardhat/testing.html).

Good points @uF4No ... On the getting-started let's not mention # of plugins anymore, let's just say "has the following plugins"

**Review by githubdoramon — APPROVED** [review]

**Review by githubdoramon — APPROVED** [review]

**Pull request merged**